### PR TITLE
List a number of usable data sources

### LINKED ISSUE #3: Find data sources for the project
Related to #1, find real data sources and write it down in a SOURCES.md file

### PR DESCRIPTION
**Pull request by Rosa-Rafe**
📖 please give me your comments on this dataset
When is finished close #3

### CODE REVIEW

File: sources.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# sources
+I will attach a dataset that I found in the wikidata
+
+[SNIRH data]https://snirh.pt/snirh/_dadosbase/site/simplex.php?OBJINFO=DADOS&FILTRA_BACIA=138&FILTRA_COVER=920123704&FILTRA_SITE=920685260
```
Comment (JJ) [inline sources.md:4]: Please add a `-` at the beginning to make it a list; also the URL must be in parentheses

File: sources.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+From SNIRH database we have data for monthly mean air temperature and monthly precipitation from 01/10/1980 to 01/02/2022.
```
Comment (JJ) [inline sources.md:6]: Try to group this paragraph with the one above. You can use a header (##) or an item list (`-`)

**Review by JJ — CHANGES_REQUESTED** [review]
It probably needs better formatting. If that's the only data source, just the changes will do.

File: sources.md (hunk 1 above)
Comment (Rosa-Rafe) [reply sources.md]: - [SNIRH data] (https://snirh.pt/snirh/_dadosbase/site/simplex.php?OBJINFO=DADOS&FILTRA_BACIA=138&FILTRA_COVER=920123704&FILTRA_SITE=920685260)
-

File: sources.md (hunk 2 above)
Comment (Rosa-Rafe) [reply sources.md]: # SNIRH database#
-  we have data for monthly mean air temperature and monthly precipitation from 01/10/1980 to 01/02/2022.
- With these data we can understand the correlation between precipitation and air temperature and answer the main question of client has.
- It seems good and we could choose the monthly average temperature and monthly precipitation value

**Comment by JJ** [comment]
Also, please remember that data sources are never 100% reliable. Either for cross-testing purposes, or to use them as alternative data sources, it would be convenient to have several data sources instead of a single one.

File: sources.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+## SNIRH database
+
+[SNIRH data](https://snirh.pt/snirh/_dadosbase/site/simplex.php?OBJINFO=DADOS&FILTRA_BACIA=138&FILTRA_COVER=920123704&FILTRA_SITE=920685260)
+
+We have data for monthly mean air temperature and monthly precipitation from 01/10/1980 to 01/02/2022.
+With these data we can understand the correlation between precipitation and air temperature and answer the main question of client has.
+
+It seems good and we could choose the monthly average temperature and monthly precipitation value
+
```
Comment (MASOTs) [inline sources.md:11]: [wordclim](https://www.worldclim.org/data/monthlywth.html)

This is another source of data on precipitation and air temperature that we can use

**Review by danielm09 — APPROVED** [review]

**Review by JJ — APPROVED** [review]
Thanks! LGTM

**Comment by Rosa-Rafe** [comment]
> what about the Fire?

no need

**Review by MarouAna — COMMENTED** [review]
Very good

**Pull request merged**